cmd/main.go

```diff
@@ -51,6 +53,7 @@ func init() {
 	targets = stringFlag(flag.CommandLine, "targets", "", "comma separated list of targets e.g. http://localhost:8081/metrics,http://localhost:8082/metrics or url1=http://localhost:8081/metrics,url2=http://localhost:8082/metrics for custom label values")
 	targetLabelsEnabled = boolFlag(flag.CommandLine, "targets.label", true, "Add a label to metrics to show their origin target")
 	targetLabelName = stringFlag(flag.CommandLine, "targets.label.name", "ae_source", "Label name to use if a target name label is appended to metrics")
+	targetLabelRegexp = stringFlag(flag.CommandLine, "targets.label.regexp", ".+", "Regexp for match metrics label values")
```

Comment:
should this be called `targets.label.filter` or similar  so it's clear that this regex will be used to filer our metrics that do not match? Also, I'm a bit worried by the fact there is no way to limit it to a specific label. Maybe it should require a format like [label]=[regex] so that specific labels can be targeted?

Reply:
Good point, I will try to create format like [label]=[regex]

Reply:
I think the best way, to use command line arg targets.label.filter as json, example:

```
-targets.label.filter '{"code":"^srv$","code2":"^name$"}'
```

So in this case we will filter multiply labels, each with own regexp.

Also default value for targets.label.filter is empty, so not use regexp in this case.

What you think? If this is ok, I will try to do this part.

Reply:
yeah I don't mind this solution. Although it would be nice if the args were consistent (i.e. the targets should arguably have the same format), it's not a big issue for me.